nncf/config/config.py

```diff
@@ -67,6 +66,15 @@ def has_extra_struct(self, struct_cls: Type[NNCFExtraConfigStruct]) -> NNCFExtra
     def get_all_extra_structs_for_copy(self) -> List[NNCFExtraConfigStruct]:
         return list(self.__nncf_extra_structs.values())
 
+    def get_redefinable_global_param_value_for_algo(self, param_name: str, algo_name: str) -> Optional:
```

Comment:
Could you please specify docstring for this method? It's not clear for me why this method is called in such way and what it doing

Reply:
Done

Reply:
Thanks!